docs/pages/repo/docs/core-concepts/monorepos/task-graph.mdx

```diff
@@ -0,0 +1,95 @@
+---
+title: The Task Graph
+description: Turborepo builds a Task Graph based on your configuration and repository structure.
+---
+
+# The Task Graph
+
+In [the previous section][3], we talked about how Turborepo uses `turbo.json` to express
+how tasks relate to each other. You can think of these relationships as dependencies between
+tasks, but we have a more formal name for them: the task graph.
```

Comment:
Let's capitalize Task Graph if we're calling it "the"! Really make it concrete, proper noun for the context of Turbo.